[2024-09-14T14:12:31.608+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-14T14:12:31.627+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v1.postgres_to_s3 manual__2024-09-14T13:14:43.254219+00:00 [queued]>
[2024-09-14T14:12:31.635+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks_v1.postgres_to_s3 manual__2024-09-14T13:14:43.254219+00:00 [queued]>
[2024-09-14T14:12:31.636+0000] {taskinstance.py:2856} INFO - Starting attempt 9 of 14
[2024-09-14T14:12:31.651+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2024-09-14 13:14:43.254219+00:00
[2024-09-14T14:12:31.662+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks_v1', 'postgres_to_s3', 'manual__2024-09-14T13:14:43.254219+00:00', '--job-id', '739', '--raw', '--subdir', 'DAGS_FOLDER/postgres_hooks_dag.py', '--cfg-path', '/tmp/tmpfsebgs5f']
[2024-09-14T14:12:31.663+0000] {standard_task_runner.py:105} INFO - Job 739: Subtask postgres_to_s3
[2024-09-14T14:12:31.663+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=65864) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-14T14:12:31.664+0000] {standard_task_runner.py:72} INFO - Started process 65865 to run task
[2024-09-14T14:12:31.705+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_hooks_v1.postgres_to_s3 manual__2024-09-14T13:14:43.254219+00:00 [running]> on host 01a3c7586b71
[2024-09-14T14:12:31.780+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks_v1' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-14T13:14:43.254219+00:00' AIRFLOW_CTX_TRY_NUMBER='9' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-14T13:14:43.254219+00:00'
[2024-09-14T14:12:31.781+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-14T14:12:31.801+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-14T14:12:31.825+0000] {postgres_hooks_dag.py:29} INFO - Saved orders data in text file get_orders.txt
[2024-09-14T14:12:31.826+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-14T14:12:31.836+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-14T14:12:31.837+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks_v1, task_id=postgres_to_s3, run_id=manual__2024-09-14T13:14:43.254219+00:00, execution_date=20240914T131443, start_date=20240914T141231, end_date=20240914T141231
[2024-09-14T14:12:31.879+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-14T14:12:31.897+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-14T14:12:31.900+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
